This diff is to remove a version check that is causing trouble with
RH's perl 5.8.1 prerelease.  Since we mark it as 5.8.0, this confuses
modules that do checks.
--- DBI-1.37/DBI.xs.prever	2003-07-07 08:42:08.000000000 -0400
+++ DBI-1.37/DBI.xs	2003-07-07 08:42:31.000000000 -0400
@@ -54,10 +54,6 @@
 #define aTHXo_
 #endif
 
-#if (PERL_VERSION < 8) || ((PERL_VERSION == 8) && (PERL_SUBVERSION == 0))
-#define DBI_save_hv_fetch_ent
-#endif
-
 
 static imp_xxh_t *dbih_getcom	   _((SV *h));
 static imp_xxh_t *dbih_getcom2	   _((SV *h, MAGIC **mgp));
